### ISSUE TITLE
How to uninstall after building with cmake on Debian?

### ISSUE THREAD

**Issue by mYnDstrEAm** [mixxxdj/mixxx#12248]
### Bug Description

I'd like to do a very simple common thing: uninstall Mixxx on Debian after installing it exactly [as described and recommended in the Compilation guide](https://github.com/mixxxdj/mixxx/wiki/Compiling-On-Linux) as building from source was the only way to get a functional mixxx in Debian 9, 10 and 11 (until Debian 12 was released).

Now that Debian 12 is out and mixxx can't be started anymore due to `error while loading shared libraries: libFLAC.so.8`, I'd like to install mixx from the Debian repos (which finally has a version that is fully functional) and uninstall the built one. How can this be done? The guide only has uninstallation info for scons `scons -c prefix=INSTALLATION_DIRECTORY install` which doesn't work and neither does `cmake uninstall`/`make uninstall`.

I think this is a bug since information how to uninstall is missing in the Wiki guide.

### Version

2.3

### OS

Debian 12

**Comment by mYnDstrEAm** [comment]
Thank you! However, this issue is not closed and yes this definitely needs to be in the guide if that's the way or most likely way to uninstall. Could you please add it there in some way?

I tried `cat install_manifest.txt | sudo xargs rm` after `cd`ing into the `mixxx/build` dir and it seems to have worked but the /usr/local/share/mixxx folder is not deleted and contains some files that probably should get deleted if not entirely (I simply moved the folder or run `sudo rm -rf /usr/local/share/mixxx`). I found out because the outputs include errors like `rm: cannot remove /usr/local/share/mixxx/skins/Deere is a directory`.

I think an uninstall option would be needed but now that it can probably in nearly all cases be installed from an installer or a repo, it's not as important anymore as it used to be...in any case until such an option is available instructions how to uninstall should be in the guide (or linked to from there). Installed the version in Debian's repos now and that worked well.

**Comment by ronso0** [comment]
I'm not sure whether an `uninstall` option should be added, hence reopening this issue.

FYI you can also pack a `deb` for manual install, sharing and later use with `cpack -G DEB`.
That could also be added to the wiki.

**Comment by FrankwaP** [comment]
Hi!

Just in case… the `cat install_manifest.txt | sudo xargs rm` mentioned previously will fail for files having spaces in their names.  Here's a more robust solution:  `cat install_manifest.txt | while read file; do sudo rm -rf ${file}; done`.

**Comment by FrankwaP** [comment]
> FYI you can also pack a `deb` for manual install, sharing and later use with `cpack -G DEB`.
> That could also be added to the wiki.

Yes I replaced the `make install` step with `cpack`+`dpkg -i`, and it does seem like a better option! Thank you for this tip!

In case other n00bs are interested, I've saved the steps in my `mixxx-utils` repo: https://github.com/FrankwaP/mixxx-utils/blob/main/install_Mixxx_with_libKeyFinder.md

BTW I build Mixxx so I could use `libKeyFinder` on Ubuntu 22. Is there a way to share the resulting `deb` ?

**Comment by daschuer** [comment]
One way would be to set up a launchpad PPA for this. 
What is missing in our ~mixxx PPA for this?

**Comment by FrankwaP** [comment]
> What is missing in our ~mixxx PPA for this?

Not sure I understand your question… but I'll still try to answer :-p 

I had Mixxx installed from `https://ppa.launchpadcontent.net/mixxx/mixxx/ubuntu` (I suppose, since I found this in my `apt/sources.list.d`), and it did not propose to detect the track keys using Key Finder (`libKeyFinder`). I saw that I needed to compile form sources.
